src/app/domain/geolocation/tracking/tracking.js

```diff
@@ -160,10 +160,10 @@ const prepareMotionData = async ({ locations, lastUploadedPoint }) => {
         // Force a stop/start transition when the temporal delta is too high
         // Note forcing the transition won't automatically create a new trip, typically a long-distance
         // flight of 12h+ should behave as one trip, even with this transition.
-        await addStopTransitions(contentToUpload, getTs(previousPoint) + 1, {
+        await addStopTransitions(contentToUpload, getTs(previousPoint) + 0.01, {
```

Comment:
nit: would be great to extract all those `0.01` values into a named constant. I think this would help undestanding what happens here without having to rely on the commit message

Reply:
Ah, I see you point and it makes sense. But, in my modest experience, I'm not a big fan of named constants in such situations, because in my opinion, it's quite less readable. 
I prefer to read `stuff + 0.01` rather than `stuff + TRANSITION_TIME_SHIFT_1_MS` typically. 
But it has the benefit of adding some doc on `TRANSITION_TIME_SHIFT_1_MS` so it might be worth it

Reply:
I added the const